src/net/sourceforge/kolmafia/textui/parsetree/ProxyRecordValue.java

```diff
@@ -662,7 +662,8 @@ public Value get_noob_skill() {
      * @return The Skill granted
      */
     public Value get_skill() {
-      String skillName = Modifiers.getStringModifier("Item", (int) this.contentLong, "Skill");
+      String skillName =
+          Modifiers.getStringModifier(ModifierType.ITEM, (int) this.contentLong, "Skill");
```

Comment:
This file is missing an import for ModifierType, hence test failures. Looks like your latest commit lost this when you replaced the wildcard.

Reply:
Fixed.